lang/lpython/Portfile

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               compiler_blacklist_versions 1.0
+PortGroup               github 1.0
+
+github.setup            lcompilers lpython 0.20.0 v
+revision                0
+categories              lang python
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 BSD
+description             High-performance typed Python compiler
+long_description        LPython aggressively optimizes type-annotated Python code. \
```

Comment:
Thanks for working on this.

84 characters > 80 characters. Please do clean this up.